From 0d43a0131453a6d3d7da7f2893938494a9db2ff4 Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Thu, 15 Nov 2018 12:16:36 -0500
Subject: [PATCH] Fix: typo 'leat' -> 'least'
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
Signed-off-by: Jérémie Galarneau <jeremie.galarneau@efficios.com>
---
 src/common/defaults.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/common/defaults.c
+++ b/src/common/defaults.c
@@ -118,7 +118,7 @@
 	}
 
 	if (rlim.rlim_max > 0 && selected_ss > rlim.rlim_max) {
-		WARN("Your system's stack size restrictions (%zu bytes) may be too low for the LTTng daemons to function properly, please set the stack size limit to at leat %zu bytes to ensure reliable operation",
+		WARN("Your system's stack size restrictions (%zu bytes) may be too low for the LTTng daemons to function properly, please set the stack size limit to at least %zu bytes to ensure reliable operation",
 			(size_t) rlim.rlim_max, (size_t) DEFAULT_LTTNG_THREAD_STACK_SIZE);
 		selected_ss = (size_t) rlim.rlim_max;
 	}
