Fix typo 'occured' -> 'occurred', submited upstream.

--- a/src/bin/lttng-sessiond/rotation-thread.c
+++ b/src/bin/lttng-sessiond/rotation-thread.c
@@ -385,7 +385,7 @@
 				&exists_status);
 		if (ret) {
 			pthread_mutex_unlock(socket->lock);
-			ERR("Error occured while checking rotation status on consumer daemon");
+			ERR("Error occurred while checking rotation status on consumer daemon");
 			goto end;
 		}
 
@@ -414,7 +414,7 @@
 				&exists_status);
 		if (ret) {
 			pthread_mutex_unlock(socket->lock);
-			ERR("Error occured while checking rotation status on consumer daemon");
+			ERR("Error occurred while checking rotation status on consumer daemon");
 			goto end;
 		}
 
--- a/src/bin/lttng-sessiond/cmd.c
+++ b/src/bin/lttng-sessiond/cmd.c
@@ -4983,7 +4983,7 @@
 
 	switch (rotation_state) {
 	case LTTNG_ROTATION_STATE_NO_ROTATION:
-		DBG("Reporting that no rotation has occured within the lifetime of session \"%s\"",
+		DBG("Reporting that no rotation has occurred within the lifetime of session \"%s\"",
 				session->name);
 		goto end;
 	case LTTNG_ROTATION_STATE_EXPIRED:
